contrib/ab-simulation/simulation.py

```diff
@@ -0,0 +1,187 @@
+# A Python 2.7 library to simulate AB tests and analyze results.
+
```

Comment:
@dillard Please add the copyright header to the python script as this is in the open source repo

###############################################################################
# Copyright 2016 Intuit
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
###############################################################################